elliott/elliottlib/bzutil.py

```diff
@@ -620,6 +644,9 @@ class JIRABugTracker(BugTracker):
     field_release_blocker = 'customfield_12319743'  # "Release Blocker"
     field_blocked_reason = 'customfield_12316544'  # "Blocked Reason"
     field_severity = 'customfield_12316142'  # "Severity"
+    field_cve_id = 'customfield_12324749'  # "CVE ID"
+    field_cve_upstream_component = 'customfield_12324751'  # "Upstream Affected Component"
```

Comment:
should this be "downstream affected component" field instead? see https://issues.redhat.com/browse/OCPBUGS-37655 upstream is `ncurses`, downstream is `rhcos`

Reply:
updated